geomesa-spark/geomesa-spark-accumulo/src/test/scala/org/locationtech/geomesa/spark/accumulo/SparkSQLDataTest.scala

```diff
@@ -0,0 +1,140 @@
+/***********************************************************************
+* Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Apache License, Version 2.0
+* which accompanies this distribution and is available at
+* http://www.opensource.org/licenses/apache2.0.php.
+*************************************************************************/
+
+package org.locationtech.geomesa.spark.accumulo
+
+import java.util.{Map => JMap}
+
+import com.vividsolutions.jts.geom.{Coordinate, Point}
+import org.apache.accumulo.minicluster.MiniAccumuloCluster
+import org.apache.spark.sql.{DataFrame, SQLContext, SparkSession}
+import org.geotools.data.DataStoreFinder
+import org.geotools.geometry.jts.JTSFactoryFinder
+import org.junit.runner.RunWith
+import org.locationtech.geomesa.accumulo.AccumuloProperties.AccumuloQueryProperties
+import org.locationtech.geomesa.accumulo.data.AccumuloDataStore
+import org.locationtech.geomesa.index.conf.QueryProperties
+import org.locationtech.geomesa.utils.text.WKTUtils
+import org.specs2.mutable.Specification
+import org.specs2.runner.JUnitRunner
+
+@RunWith(classOf[JUnitRunner])
+class SparkSQLDataTest extends Specification {
+  val createPoint = JTSFactoryFinder.getGeometryFactory.createPoint(_: Coordinate)
+
+  "sql data tests" should {
+    sequential
+
+    System.setProperty(QueryProperties.SCAN_RANGES_TARGET.property, "1")
+    System.setProperty(AccumuloQueryProperties.SCAN_BATCH_RANGES.property, s"${Int.MaxValue}")
```

Comment:
should probably put these into the 'before' step, and also need to clear them into an 'after' step